app/experimenter/nimbus-ui/src/lib/experiment.ts

```diff
@@ -5,37 +5,32 @@
 import { RedirectCheck } from "../components/AppLayoutWithExperiment";
 import { getAllExperiments_experiments } from "../types/getAllExperiments";
 import { getExperiment_experimentBySlug } from "../types/getExperiment";
-import { NimbusExperimentStatus } from "../types/globalTypes";
+import {
+  NimbusExperimentPublishStatus,
+  NimbusExperimentStatus,
+} from "../types/globalTypes";
 
 export function getStatus(
   experiment?: getExperiment_experimentBySlug | getAllExperiments_experiments,
 ) {
   const status = experiment?.status;
-
-  // The experiment is in review or accepted state
-  const preparation = [
-    NimbusExperimentStatus.REVIEW,
-    NimbusExperimentStatus.ACCEPTED,
-  ].includes(status!);
+  const publishStatus = experiment?.publishStatus;
 
   // The experiment is or was out in the wild (live or complete)
   const released = [
     NimbusExperimentStatus.LIVE,
     NimbusExperimentStatus.COMPLETE,
   ].includes(status!);
```

Comment:
Oh released means it was launched I see....  I've always used the word 'launched' for that, we might want to standardize on some terminology here.

Reply:
I'm cool with renaming this to `launched`, then `AppLayoutSidebarReleased` (previously "Locked") can be called `AppLayoutSidebarLaunched`. Naming here has definitely been an iterative process since we needed more states before, good to have a fresh pair of eyes on it. 😛

Reply:
Renamed!